### PR TITLE
chore: add issue templates

### PR DESCRIPTION
**Pull request by luke-biel**

### CODE REVIEW

**Review by frondeus — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+name: Feature request
+about: Suggest an idea for this project
+title: Feature
+labels: enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (Kononnable) [inline .github/ISSUE_TEMPLATE/feature_request.md:11]: Is any feature request not related to a problem? :smile:

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Bug report
+about: Short description
+title: Bug
+labels: bug
```
Comment (Kononnable) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]: Should labels be added automatically, or should we label the issues once we make sure it's really a bug, feature request, not someone not knowing how to use CDL/how it's designed.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+name: Bug report
+about: Short description
+title: Bug
```
Comment (Kononnable) [inline .github/ISSUE_TEMPLATE/bug_report.md:4]: Unless we want to use some specific format I think we can probably leave it empty.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (luke-biel) [reply .github/ISSUE_TEMPLATE/feature_request.md:11]: IDK, this is default template

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (luke-biel) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Removed

**Review by USSAMOH2 — APPROVED** [review]

**Pull request merged**